client/app/core/navigation.service.js

```diff
@@ -1,12 +1,22 @@
+import NavActions from '../actions/nav'
+/* eslint no-unused-vars: 0 */
```

Comment:
Looks like all the vars are used we should not need this, maybe I'm missing something 🤔

Reply:
Just took care of removing the unsubscribe variable which went ahead and removed this line.